recipes-support/blink-led/blink-led/blink-led

```diff
@@ -20,3 +20,13 @@ elif rauc status | grep -q 'Booted from: rootfs.0 (system0)' &&
     echo 0 > /sys/class/leds/led-red/brightness
     echo heartbeat > /sys/class/leds/led-green/trigger
 fi
+rauc_status_old=$(rauc status)
+
+while true; do
+    rauc_status_new=$(rauc status)
+    if [[ "$rauc_status_old" != "$rauc_status_new" ]]
```

Comment:
The builtin `[[` is not available in POSIX `sh`. This requires something like BASH. Please use `[` instead.